tests/cdi/cdi_test.go

```diff
@@ -0,0 +1,135 @@
+package cdi_test
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+// template parameters
+const (
+	pvcEPHTTPNOAUTHURL  = "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
```

Comment:
Would be cool if we could override the VM image with an environment variable but still use this as a default when nothing else has been specified.

Reply:
would address #300